src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+[[I nodded sagely, absorbing Belinda’s wisdom about Ron’s uselessness|The Gutter - Bar - Belinda]].
+
+::The Gutter - Bar - Belinda ron
+Belinda chuckled. “That stupid piece of shit? One time he lorded a fresh puzzle book over me for days. Didn’t bother me a bit though. I knew he couldn’t resist a pool game, so I just waited until he got shithoused and challenged him. Honestly, I hope he finds another book soon, because I think he forgot it even happened.”
+
+[[Belinda made a good point. Ron is a stupid piece of shit|The Gutter - Bar - Belinda]].
+
+::The Gutter - Bar - Death
+I did one joke too many and I died like an idiot. Belinda blew my brains across the bar. Ron laughed. Ugh.
+
+As my spirit approached the outskirts of the Dark Carnival, a hooded figure came out to greet me. He pulled his hood down. It was a skeleton of Shaggy 2 Dope and he had a message:
+
+“You dumbass bitch. This isn’t how the game ends! Get back in there and don’t go doin’ stupid shit again.”
+
+The Shaggy skeleton made the shocker with his bone hands.
+
+[[I felt time swirl and morph around me|The Gutter - Bar - Belinda]].
+
+::The Gutter - Bar - Darryl
+I sat down next to Darryl. He was a dipshit, but at least he wasn’t loud about it. He also didn’t care for Ron, which to me qualified as having perfect judgment. I started right in:
+
+>>[[“What’s the deal with Toe? He not talking today?”|The Gutter - Bar - Darryl game]]
+>>[[“What do you dream about, Darryl?"|The Gutter - Bar - Darryl dream]]
+>>[[“Is Toe his real name?”|The Gutter - Bar - Darryl toe]]
+>>[[“Why don’t you like Ron?”|The Gutter - Bar - Darryl ron]]
+>>[["See ya, Darryl."|The Gutter - Bar hub]]
+
+::The Gutter - Bar - Darryl game
+Darryl shook his head. “Eh, he never says much. About the only thing he ever does is pour and play games.”
+
+“What kind of games?” I asked.
+
+“Oh, anything. He had a collection, but some little shits recently stole most of it. It’s gotten to the point where he’s only got the stuff for pool and he won’t give it up to anybody.”
+
+[[I took note of Darryl’s observations|The Gutter - Bar - Darryl]].
+
+::The Gutter - Bar - Darryl dream
+Darryl’s eyes widened and he gulped. He let his vision linger on the edge of the bar for a moment before he spoke.
+
+“Mostly Grimace, the dark purple beast spoken of in the Book of Revelations that you would get with every McDonalds happy meal. When Grimace comes on his dark chariot driven by McNuggets, those of us who haven’t supped of the fry and the flesh weekly will be punished. It will not be swift. It will be the opposite of mercy.”
+
+Darryl cleared his throat, as if he’d just spoken several sentences by accident.
+
+“Oh, and last night I had a sexy dream about the mom from Family Guy. Don’t tell nobody.”
```

Comment:
Maybe we should italicize _Family Guy_?

Reply:
I dunno, I kinda like it this way too. idk

Reply:
I don't want the Family Guy to get angry at us and our game. I italicized.